paymaster-analysis/.env.example

```diff
@@ -0,0 +1,46 @@
+# Dummy ETH Address Private Key
+PRIVATE_KEY = "0x..." # Add "0x" to Private Key if not already added.
+
+# Pimlico Values
+
+# Alchemy Values (At the time of writing this, Pimlico doesn't support Sepolia, so using Goerli.)
+PIMLICO_CHAIN = "goerli"
+PIMLICO_CHAIN_ID = "5"
+PIMLICO_RPC_URL = "https://rpc.ankr.com/eth_goerli"
+PIMLICO_API_KEY = "" # https://dashboard.pimlico.io/apikeys
+PIMLICO_ENTRYPOINT_ADDRESS = "0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789"
+PIMLICO_MULTISEND_ADDRESS = "0x38869bf66a61cF6bDB996A6aE40D5853Fd43B526"
+PIMLICO_ERC20_PAYMASTER_ADDRESS = "0xEc43912D8C772A0Eba5a27ea5804Ba14ab502009"
+PIMLICO_USDC_TOKEN_ADDRESS = "0x07865c6E87B9F70255377e024ace6630C1Eaa37F" # https://usdcfaucet.com/
+# Make sure all nonce are unique for it to deploy account when run initially.
+PIMLICO_ACCOUNT_NONCE = "1"
+PIMLICO_ERC20_NONCE = "2"
+PIMLICO_ERC721_NONCE = "3"
+
+# Alchemy Values (If you want to use goerli, use goerli chain id and corresponding API key, Gas Policy, etc.)
+ALCHEMY_CHAIN = "sepolia" # or "goerli"
+ALCHEMY_CHAIN_ID = "11155111"
+ALCHEMY_RPC_URL = "https://eth-sepolia.g.alchemy.com/v2/Your_API_Key"
+ALCHEMY_API_KEY = "" # https://dashboard.alchemy.com/apps
+ALCHEMY_GAS_POLICY = "" # https://dashboard.alchemy.com/gas-manager
+ALCHEMY_ENTRYPOINT_ADDRESS = "0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789"
+ALCHEMY_MULTISEND_ADDRESS = "0x38869bf66a61cF6bDB996A6aE40D5853Fd43B526"
+# Make sure all nonce are unique for it to deploy account when run initially.
+ALCHEMY_ACCOUNT_PAYMASTER_NONCE = "4"
+ALCHEMY_ERC20_PAYMASTER_NONCE = "5"
+ALCHEMY_ERC721_PAYMASTER_NONCE = "6"
+
+# Safe Values
+SAFE_VERSION = "1.4.1"
+
+# Token Operation Values (Based on the chain selected, these values should also change accordingly.)
+PIMLICO_ERC20_TOKEN_CONTRACT = ""
+PIMLICO_ERC721_TOKEN_CONTRACT = ""
+ALCHEMY_ERC20_TOKEN_CONTRACT = ""
+ALCHEMY_ERC721_TOKEN_CONTRACT = ""
```

Comment:
Nit: would include this in the respective bundler values since many similar variables are parts of it

Reply:
Makes sense 👍🏾

Reply:
Done 👍🏾